docs/docs/general/client.md

```diff
@@ -57,15 +57,15 @@ Global guards, it's an array of functions
 
 ### botGuilds
 
-The application commands are executed only on this list of guilds by default, ex. slash, button, selectmenu, contextmenu
+The application commands are executed only on this list of guilds by default, ex. slash, button, select-menu, contextmenu
 
 | type       | default | required |
 | ---------- | ------- | -------- |
 | IGuild [ ] | [ ]     | No       |
 
 ## Intents
 
-You must specify the "**intents**" of your bot when you initialize the Client, it specify which informations your bot receive from the Discord's servers, **it's different from the permissions**
+You must specify the "**intents**" of your bot when you initialize the Client, it specify which information's, your bot receive from the Discord's servers, **it's different from the permissions**
```

Comment:
Should be "information"?

Reply:
multiple intent allows multiple information's, hence it should be ``information's``

Reply:
refactored, review again.